pastepwn/core/paste.py

```diff
@@ -15,27 +15,13 @@ def __init__(self, key, title, user, size, date, expire, syntax, scrape_url, ful
         self.full_url = full_url
         self.body = None
 
-    def set_body(self, body):
-        if body is None:
-            body = ""
```

Comment:
That's not exactly the same. Could you please change it to:
```python
def set_body(self, body=""):
    self.body = str(body or "")
```

Reply:
Oh, forgot to account for None and other falsy values